doc/modules/cross_validation.rst

```diff
@@ -534,19 +534,32 @@ Stratified k-fold
 folds: each set contains approximately the same percentage of samples of each
 target class as the complete set.
 
-Example of stratified 3-fold cross-validation on a dataset with 10 samples from
-two slightly unbalanced classes::
+Here is an example of stratified 3-fold cross-validation on a dataset with 50 samples from
+two unbalanced classes.  We show the number of samples in each class and compare with 
+:class:`KFold`.
 
-  >>> from sklearn.model_selection import StratifiedKFold
+  >>> from sklearn.model_selection import StratifiedKFold, KFold
+  >>> import numpy as np
 
-  >>> X = np.ones(10)
-  >>> y = [0, 0, 0, 0, 1, 1, 1, 1, 1, 1]
-  >>> skf = StratifiedKFold(n_splits=3)
-  >>> for train, test in skf.split(X, y):
-  ...     print("%s %s" % (train, test))
-  [2 3 6 7 8 9] [0 1 4 5]
-  [0 1 3 4 5 8 9] [2 6 7]
-  [0 1 2 4 5 6 7] [3 8 9]
+  >>> X, y = np.ones((50, 1)), np.hstack(([0] * 45, [1] * 5))
+
```

Comment:
don't skip lines (here and above) or this will be shown as different code blocks

Reply:
Thanks for the inputs. I was confused previous because different examples in the same documentation had different formatting (meaning there were some examples with sip lines inbetween them).